wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/BaseCallParticipantView.swift

```diff
@@ -224,7 +224,11 @@ class BaseCallParticipantView: OrientableView, AVSIdentifierProvider {
     private func updateFillMode() {
         // Reset scale if the view was zoomed in
         scalableView?.resetScale()
-        videoView?.shouldFill = shouldFill
+        updateVideoShouldFill(shouldFill)
+    }
+
+    func updateVideoShouldFill(_ shouldFill: Bool) {
+        // TODO: override in subsclasses
```

Comment:
```suggestion
        fatalError("override in subsclass")
```

Reply:
I had this before, but some unit tests uses this base view through several layers 🤷‍♂️ So I turned it into a comment..